[
    {
        "id": 298028063,
        "name": "CVE-2014-0472",
        "full_name": "christasa\/CVE-2014-0472",
        "owner": {
            "login": "christasa",
            "id": 35037256,
            "avatar_url": "https:\/\/avatars.githubusercontent.com\/u\/35037256?v=4",
            "html_url": "https:\/\/github.com\/christasa"
        },
        "html_url": "https:\/\/github.com\/christasa\/CVE-2014-0472",
        "description": "CVE-2014-0472 Django unexpected code execution using reverse()",
        "fork": false,
        "created_at": "2020-09-23T16:21:57Z",
        "updated_at": "2022-10-05T07:27:06Z",
        "pushed_at": "2020-12-24T08:12:32Z",
        "stargazers_count": 4,
        "watchers_count": 4,
        "forks_count": 0,
        "allow_forking": true,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": [],
        "visibility": "public",
        "forks": 0,
        "watchers": 4,
        "score": 0
    }
]